src/contrib/persistence/Akka.Persistence.Query.Sql/AllPersistenceIdsPublisher.cs

```diff
@@ -175,16 +175,12 @@ private bool Active(object message)
         {
             switch (message)
             {
-                case CurrentPersistenceIds added:
```

Comment:
CurrentPersistenceIds message will only be received in the `Waiting` method (previously called `Initializing`)